v1.0/electrodeContact.schema.json

```diff
@@ -0,0 +1,56 @@
+{
+  "_type": "https://openminds.ebrains.eu/sands/ElectrodeContact",
+  "description": "Structured information describing the contact point of an electrode.",
+  "required": [
+    "contactPoint",
+    "identifier"
+  ],
+  "properties": {
+    "contactPoint": {
+      "_instruction": "Add the central coordinate of this electrode contact point.",
+      "_linkedTypes": [
+        "https://openminds.ebrains.eu/sands/CoordinatePoint"
+      ]
+    },
+    "definedIn":{
+      "type": "array",
+      "_instruction": "Add one or several files in which the electrode contact is defined in.",
+      "_linkedTypes": [
+        "https://openminds.ebrains.eu/core/FileInstance"
+      ],
+      "minItems": 1,
+      "uniqueItems": true
+    },
+    "internalIdentifier": {
+      "type": "string",
+      "_instruction": "Enter the identifier used for this electrode contact within the file it is stored in."
+    },
+    "relatedRecordings":{
+      "type": "array",
+      "_instruction": "Add one or several files in which the recorded data from this electrode contact were stored.",
+      "_linkedTypes": [
+        "https://openminds.ebrains.eu/core/FileInstance"
+      ],
+      "minItems": 1,
+      "uniqueItems": true
+    },
+    "relatedStimulations":{
+      "type": "array",
+      "_instruction": "Add one or several files in which the stimulation of this electrode contact were stored.",
```

Comment:
"Add one or several files in which the stimulations applied via this electrode contact were stored."